4001-5000/LIT4145Onausp.xml

```diff
@@ -4,7 +4,13 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
    <teiHeader xml:base="https://betamasaheft.eu/">
       <fileDesc>
          <titleStmt>
-            <title xml:id="t1">On auspicious and inauspicious days</title>
+            <title xml:id="t1" xml:lang="gez">ሐሳበ፡ ጽንዕት፡ ወጽንጽንት፡</title>
+            <title corresp="#t1" xml:lang="gez">Ḥasāba ṣǝnǝʿt wa-ṣǝnṣǝnt</title>
+            <title corresp="#t1" xml:lang="en">Calculation of auspicious and inauspicious (days)</title>
+            <title corresp="#t1" xml:lang="fr">Calculation sur les (jours) fastes et néfastes</title>
```

Comment:
I wonder if we need the French title here. It does not help, and it invites to using other European languages than English, which will increase the difficulty of using the database. I will write an issue.

Reply:
I used the french title, because the primary secondary source on this work is by S. Grébaut: "Les jours fastes et néfastes d’après le ms. éthiopien n° 3 de M. E. Delorme". Some may know it under this title.

Reply:
Me to, but many people have never came to check what is "fastes et néfastes". If you can make an apt translation into English, catching the connotation, this would be really helpful. Otherwise there are also plenty of Ethiopic titles in English and Italian that can be included; but our task it actually to make descriptions in English.

Reply:
OK. I will delete the French title translation.